src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Display = ({ displayValue }) => (
+  <div>
+    {displayValue || 0}
+  </div>
+);
+
+Display.propTypes = {
+  displayValue: propTypes.string.isRequired,
+};
```

Comment:
- [ ] OPTIONAL - Rather than this: `displayValue || 0`, consider doing this instead:
```
Display.defaultProps = {
  displayValue: '0'
}

Display.propTypes = {
  displayValue: propTypes.string,
}
```
It's a much neater syntax.